source/clear-linux/concepts/telemetry-about.rst

```diff
@@ -0,0 +1,76 @@
+.. _telemetry-about:
+
+Telemetrics
+###########
+
+Telemetry is one of the key features of |CLOSIA|. The |CL| team is focused on
+the quality of the telemetry data, and is moving from issues that are detected
+and reported by users to issues detected and repaired by providers. Ideally,
+these issues will be undetected by users.
+
+On the |CL| team, we refer to *telemetrics*, which is a combination word made from:
+
+*	Telemetry which is sensing and reporting data.
+*	Analytics which is using visualization and statistical inferencing to make
+	sense of the reported data.
+
+|CL| telemetry reports system-level debug/crash information using specialized probes. The
+probes monitor system tasks such as :abbr:`swupd (software updater)`, kernel
+oops, machine error checks, and BIOS error report table for unhandled hardware
+failures. Telemetry enables real-time issue reporting to allow system
+developers to quickly focus on an issue and monitor corrective actions.
+
+|CL| telemetry is fully customizable and can be used during software development
+for debugging purposes. You can use libtelemetry in your code to create custom
+telemetry records. You can also use telem-record-gen in script files or call
+it from another program.
+
+Architecture
+************
+
+|CL| telemetry has two fundamental components, which are shown in figure 1:
+
+*	Client:  generates and delivers records to the backend server via the network.
+*	Backend: captures records sent from the client and displays the cumulative
+	content through a specialized interface.
+
+Note: To capture records for analysis, you must set up your own backend server.
```

Comment:
For this note, I would say something like "If you want to capture your own records for analysis, you will need to set up your own backend server"